hw6_social.py

```diff
@@ -192,6 +319,19 @@ def graphTopNStates(stateCounts, stateFeatureCounts, n, title):
 Returns: None
 '''
 def graphRegionComparison(regionDicts, title):
+    featurelst=[]
+    regionlst=[]
+    regionvalue=[]
+    for i in regionDicts:
+        templist=[]
+        x=regionDicts[i]
+        for j in x:
+            if j not in featurelst:
+                featurelst.append(j)
+            templist.append(x[j])
+        regionvalue.append(templist)
+        regionlst.append(i)
+    sideBySideBarPlots(featurelst, regionlst, regionvalue, title)
     return
 
 

```

Comment:
**graphHashtagSentimentByFrequency**  function is empty

Reply:
Can you please check now